builtin/logical/cassandra/path_roles.go

```diff
@@ -54,6 +54,11 @@ template values are '{{username}}' and
 				Default:     "4h",
 				Description: "The lease length; defaults to 4 hours",
 			},
+			
+			"consistency": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "The consistency level for the operations.",
```

Comment:
Will do. Based on my research, the default in gocql is "Quorum". I read that as we don't have to set the default. Reasonable?